web-app/js/portal/cart/DownloadPanelItemTemplate.js

```diff
@@ -176,7 +176,7 @@ Portal.cart.DownloadPanelItemTemplate = Ext.extend(Ext.XTemplate, {
 
     _removeButtonOnClick: function(button) {
         var collectionId = this.getIdFromButtonContainerId(button, "removeButtonId");
-        var record = this.dataCollectionStore.getRecordFromUuid(collectionId);
+        var record = this.dataCollectionStore.dataCollectionWithUuid(collectionId);
```

Comment:
Given the DataCollectionStore only holds DataCollections, and there are other get methods already (`getAt(...)`, `getById(...)`) could I simply call it `getByUuid(...)`?